src/UrlSession.php

```diff
@@ -15,9 +15,6 @@ class UrlSession
      */
     public function handle($request, Closure $next)
     {
-        ini_set('session.use_trans_sid', 1);
```

Comment:
maybe we should add a comment 

    // intentionally left empty - just for marking the requests

?

Reply:
Added